common/src/main/java/com/microsoft/identity/common/internal/authorities/Authority.java

```diff
@@ -190,7 +191,7 @@ private static Authority createAadAuthority(@NonNull final Uri authorityUri,
         return new AzureActiveDirectoryAuthority(audience);
     }
 
-    public abstract OAuth2Strategy createOAuth2Strategy();
+    public abstract OAuth2Strategy createOAuth2Strategy(@NonNull final OAuth2StrategyParameters options) throws ClientException;
```

Comment:
may be rename `options` to `parameters` or `strategyParameters` ? (here and everywhere else)

Reply:
I guess it's actually better to keep it `options` as there are already plenty of things called parameters

Reply:
Resolved in cdc3088b0cbb0204c84b6e5ebfac32797eb32613